kyaml/setters2/settersutil/fieldsetter.go

```diff
@@ -127,3 +136,30 @@ func SetAllSetterDefinitions(openAPIPath string, dirs ...string) error {
 	}
 	return nil
 }
+
+// syncOpenAPIValuesWithSchema syncs the setter values in global openAPI schema
+// with the ones in openAPIPath and writes them back
+func syncOpenAPIValuesWithSchema(openAPIPath string) error {
+	refs, err := openapi.DefinitionRefs(openAPIPath)
+	if err != nil {
+		return err
+	}
+	for _, ref := range refs {
+		sch := openapi.Schema().Definitions[ref]
+		cliExt, err := setters2.GetExtFromSchema(&sch)
+		if cliExt == nil || cliExt.Setter == nil || err != nil {
```

Comment:
Can we safely ignore all errors here?

Reply:
Yes. If we are not able to derive a value for setter refs in global schema, we should continue. But we should throw an error in line 161 as it should never happen and if it does then the problem is for entire operation etc. Will update the PR.